main.go

```diff
@@ -60,14 +63,27 @@ func exportEnvironmentWithEnvman(keyStr, valueStr string) error {
 	return cmd.Run()
 }
 
+func initCommitMessageLength() error {
+	configs, err := envman.GetConfigs()
+	if err != nil {
+		return err
+	}
+
+	maxCommitMessageLength = configs.EnvBytesLimitInKB*1024 - len(trimEnding)
```

Comment:
NOTE: it would be better if `maxCommitMessageLength` would be the max commit message length regardless of the used separator.

Reply:
Updated code.